### PR TITLE
Add AdventureSMP

### PR DESCRIPTION
**Pull request by sluhtie**
## Type of change

- [x] Add new directory for a server
- [ ] Update directory of a server
- [ ] Remove directory of a server
- [ ] Documentation Update

## Checklist

- [x] I have read the [README](https://github.com/LabyMod/server-media/blob/master/README.md) doc
- [x] I have compressed the images appropriately (e.g. on https://tinypng.com)
- [x] I have created the manifest.json with the required values

### CODE REVIEW

File: minecraft_servers/adventuresmp/manifest.json [hunk 1]
```diff
@@ -0,0 +1,25 @@
+{
+    "server_name": "adventuresmp",
+    "nice_name": "AdventureSMP",
+    "direct_ip": "adventuresmp.de",
+    "supported_languages": ["de"],
+    "social": {
+        "web_shop": "https://shop.adventuresmp.de",
```
Comment (maaaathis) [inline minecraft_servers/adventuresmp/manifest.json:7]: Not reachable

File: minecraft_servers/adventuresmp/manifest.json [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+        "discord": "https://discord.gg/E7EWKugWXR"
+    },
+    "gamemodes": {
+        "survival": {
+        "name": "Survival", 
+        "color": "#48E863"
```
Comment (maaaathis) [inline minecraft_servers/adventuresmp/manifest.json:13]:
```suggestion
            "name": "Survival", 
            "color": "#48E863"
```

File: minecraft_servers/adventuresmp/manifest.json [hunk 3]
```diff
@@ -0,0 +1,25 @@
+{
+    "server_name": "adventuresmp",
+    "nice_name": "AdventureSMP",
+    "direct_ip": "adventuresmp.de",
+    "supported_languages": ["de"],
```
Comment (maaaathis) [inline minecraft_servers/adventuresmp/manifest.json:5]:
```suggestion
    "supported_languages": [
        "de"
    ],
```

**Review by maaaathis — CHANGES_REQUESTED** [review]

File: minecraft_servers/adventuresmp/manifest.json (hunk 1 above)
Comment (sluhtie) [reply minecraft_servers/adventuresmp/manifest.json:9]: It's waiting approval, will be reachable in next 48 hours.

**Comment by sluhtie** [comment]
I removed the webshop link for now, will add later as soon It has been reviewed and confirmed from tebex

**Review by DoJapHD — APPROVED** [review]

**Pull request merged**